cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1969,27 +1969,40 @@ public NodeAddressType getPreferredNodeAddressType(GenericKafkaListener listener
         }
     }
 
-    private String generateBrokerConfiguration(boolean controlPlaneListener)   {
-        return new KafkaBrokerConfigurationBuilder()
-                .withBrokerId()
-                .withRackId(rack)
-                .withZookeeper(cluster)
-                .withLogDirs(VolumeUtils.getDataVolumeMountPaths(storage, mountPath))
-                .withListeners(cluster, namespace, listeners, controlPlaneListener)
-                .withAuthorization(cluster, authorization)
-                .withCruiseControl(cluster, cruiseControlSpec, ccNumPartitions, ccReplicationFactor, ccMinInSyncReplicas)
-                .withUserConfiguration(configuration)
-                .build().trim();
+    private String generateBrokerConfiguration(boolean controlPlaneListener, boolean escapeTheZoo)   {
+        if (escapeTheZoo)   {
+            return new KafkaBrokerConfigurationBuilder()
+                    .withNodeId()
+                    .withKraft(cluster, namespace, replicas)
```

Comment:
So currently every broker must also be a voter, right?

Reply:
Yes. Every broker is controller.